src/group_theory/conjugates.lean

```diff
@@ -0,0 +1,142 @@
+/-
+
+Author: Michael Howes and Scott Morrison, 2019
+
+The definition of conjugates in a group and the definition
+of the normal closure of a subset of a group.
+-/
+import algebra.group
+import group_theory.subgroup
+import tactic.rewrite
+
+variable {α : Type}
+variable [group α]
+
+
+
+/-- Two elements a b of a group are conjugates if there exists a
+c such that c*a*c⁻¹ = b -/
+def is_conjugate (a b : α) : Prop := ∃ c : α, c*a*c⁻¹ = b
```

Comment:
There is already an `is_conj` relation in `algebra.group`

Reply:
Thanks for pointing this out. In light of this I think it would probably best to put my contribution in `group_theory.subgroup` rather than in a new file.